src/qt/bitcoin.cpp

```diff
@@ -573,7 +573,7 @@ int main(int argc, char *argv[])
     {
         HelpMessageDialog help(NULL, mapArgs.count("-version"));
         help.showOrPrint();
-        return 1;
+        return 0;
```

Comment:
Dunno, it would make some sense to be consistent, but I don't care deeply. But if you feel like changing it, this PR would be the chance to do it.